asm/f32/dot.go

```diff
@@ -4,18 +4,18 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package asm
+package f32
 
-func DsdotUnitary(x, y []float32) (sum float64) {
+func DotUnitary(x, y []float32) (sum float32) {
```

Comment:
This is not right. DSDOT take single precision input an returns double precision.